design/one-pager-component-mutable-and-versioning.md

```diff
@@ -183,13 +184,24 @@ metadata:
   name: fancytraits.core.oam.dev
 spec:
   revision:
-    aware: true
+    alwaysControl: true
     revisonPath:
       - "spec.revision"
   definitionRef:
     name: fancytraits.core.oam.dev
 ```
 
+#### Revision field rule
+
+Versioning mechanism will be related with `revision` field.
+
+- If no traits binding with a component have revision field, OAM runtime will always update workload directly.
+- If `alwaysControl` field is true, OAM runtime will always create a new workload when component revision changed
```

Comment:
> when component revision changed

I wonder if you can clarify which component revision changed? Is it the revision in the trait or the component itself

Reply:
> If `alwaysControl` field is true, OAM runtime will always create a new workload when component ~~revision~~ changed

remove revision?
@wonderflow

Reply:
@henrywangx good idea

Reply:
> I wonder if you can clarify which component revision changed? Is it the revision in the trait or the component itself

component itself @ryanzhang-oss